cmake/CMakeLists.txt

```diff
@@ -4,39 +4,58 @@ cmake_minimum_required(VERSION 2.8)
 # Project
 project(protobuf C CXX)
 
+# CMake policies
+cmake_policy(SET CMP0022 NEW)
+
 # Options
+option(protobuf_VERBOSE "Enable for verbose output" OFF)
```

Comment:
You can see the [CMakeLists.txt](https://github.com/Kitware/CMake/blob/master/CMakeLists.txt) file of the [CMake](http://www.cmake.org) build tool. For example line [63](https://github.com/Kitware/CMake/blob/master/CMakeLists.txt#L63) option *CMake_INSTALL_COMPONENTS* of line [74](https://github.com/Kitware/CMake/blob/master/CMakeLists.txt#L74) option *CMake_INSTALL_DEPENDENCIES*. These options have been created on my initiative.
Then we will continue to use the *project_OPTION_NAME* variable naming.